bob-common/src/error.rs

```diff
@@ -38,6 +38,10 @@ impl Error {
         self.ctx == Kind::PossibleDiskDisconnection
     }
 
+    pub fn is_network_error(&self) -> bool {
+        self.ctx == Kind::Timeout || matches!(self.ctx, Kind::Failed(_))
```

Comment:
`Timeout` is not a network error! It is specific for particular request (one can timeout, the following not). 
You should check actual error code from gRPC request. Not the high-level Bob error